prisma/seeds/pullRequests.ts

```diff
@@ -0,0 +1,51 @@
+import { faker } from "@faker-js/faker";
+import { PrismaClient, PullRequest } from "@prisma/client";
+import colors from "colors";
+import { parseInt, random, sample } from "lodash";
+import { PullRequestStatusType, PullRequestType } from "./types";
+
+const prisma = new PrismaClient();
+
+colors.enable();
+
+const seedPullRequests = async () => {
+  const pullRequests: PullRequestType[] = [];
+
+  const projects = await prisma.project.findMany({
+    select: { id: true },
+  });
+
+  const users = await prisma.user.findMany({
+    select: { id: true },
+  });
+
+  console.log(`Seeding Pull Requests for ${projects.length} projects`.blue);
+
+  let prCounter = 1;
+
+  for (let i = 0; i < users.length; i++) {
+    for (let j = 0; j < projects.length; j++) {
+      pullRequests.push({
```

Comment:
Should `prCounter` be right above this line instead? Because prId is based on project, not the user.

Reply:
Okay.. got it